packages/laconia-adapter-api/src/createWebSocketAdapter.js

```diff
@@ -0,0 +1,12 @@
+const { res, parseWebSocket } = require("@laconia/event").apigateway;
+
+const createWebSocketAdapter = () => app => async (...args) => {
+  try {
+    const output = await app(parseWebSocket(...args));
```

Comment:
Continuing our previous discussion, as JavaScript arguments are flexible, I think we would be better off making it explicit here by calling it `event`. parseWebSocket is also only accepting `event` parameter.

By that I mean, it is ok for the signature to be `const createWebSocketAdapter = () => app => async (event)`, it will not break anything.

Reply:
I changed that